statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -748,117 +1072,180 @@ def forecast_interval(self, y, steps, alpha=0.05):
+        order, y_1{t-1}, ..., y_{neqs}{t-1}, ..., y_1{t-k_ar}, ...
+        y_{neqs}{t-1}
+    fittedvalues : array
+        `nobs x neqs` array of in-sample predicted values.
+    info_criteria : dict
+        A dictionary containing the information criteria values.
     k_ar : int
+        The number of lags in the model.
     k_trend : int
-    llf
-    model
-    names
+        The number of trend variables included. Equals the polynomial order
+        of a trend variable + 1.
+    llf : float
+        The log-likelihood of the fitted model.
+    model : `VARModel`
+        A reference to the `statsmodels.tsa.vector_ar.var_model.VARModel`
+        instance.
+    names : list
+        The names of the
     neqs : int
         Number of variables (equations)
     nobs : int
+        The number of observations available for estimation. This excludes
+        the first k_ar "pre-sample" observations of the original Y.
     n_totobs : int
-    params
+        The total number of observations. `k_ar` + `nobs`
     k_ar : int
         Order of VAR process
-    params : ndarray (Kp + 1) x K
-        A_i matrices and intercept in stacked form [int A_1 ... A_p]
-    pvalues
+    params : ndarray (k_trend + neqs*k_ar) x neqs
+        Trend_coefficients and A_i matrices in stacked form.
     names : list
         variables names
-    resid
-    roots : array
-        The roots of the VAR process are the solution to
-        (I - coefs[0]*z - coefs[1]*z**2 ... - coefs[p-1]*z**k_ar) = 0.
-        Note that the inverse roots are returned, and stability requires that
-        the roots lie outside the unit circle.
-    sigma_u : ndarray (K x K)
-        Estimate of white noise process variance Var[u_t]
-    sigma_u_mle
-    stderr
-    trenorder
-    tvalues
-    y :
-    ys_lagged
-    """
+    resid : ndarray
+        `nobs` x `neqs` array of residuals.
+    cov_resid : ndarray (neqs x neqs)
+        Estimate of white noise process variance Var(resid_t). This is the
+        equation by equation covariance matrix. It is the same as
+        `cov_resid_mle` * `nobs` / (`nobs` - `neqs`*`k_ar` - `k_trend`)
+    cov_resid_mle : ndarray (neqs x neqs)
+        cov_resid without a degrees of freedom adjustment. Asymptotically,
+        `cov_resid_mle` is equivalent to `cov_resid`
+    trendorder : int
+        The polynomial order of the trend. Ie., nobs**trendorder. If the
+        model is fit without a constant, then trendorder = None.
+    Y : array
+        Endogenous variable.
+    X : array
+        Lagged endogenous variables. Trend variable(s) are prepended in
+        columns. Ie., the first `k_trend` columns are the trend.
+
+    Notes
+    -----
+    """ + _var_math_doc
     _model_type = 'VAR'
 
-    def __init__(self, endog, endog_lagged, params, sigma_u, lag_order,
-                 model=None, trend='c', names=None, dates=None):
+    def __init__(self, model, Y, X, params, cov_resid, k_ar, names):
```

Comment:
large change in signature,  who not keywords, 
all required except names?
dropped keywords taken from model.attributes